### PR TITLE
Add calculator Initial Display

### PR DESCRIPTION
**Pull request by benardop**
Changes I made in this PR:
- Created the components folder
-  Added Calculator.js script file in the components folder
-  Added the calculator styles to the index.css file

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Item from './Item';
+
+function Calculator() {
```
Comment (danmainah) [inline src/components/Calculator.js]: - [ ] Kindly convert your component to a class-based component, this is a project requirement.

File: src/components/Item.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Item extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.handleObjectChange = this.handleObjectChange.bind(this);
```
Comment (danmainah) [inline src/components/Item.js:7]: - [ ] [optional] I encountered the error below while running your app seems like you initialized a prop that you never used.
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @benardop , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
- [ ] Kindly ensure your readme file has instructions on how to run the project locally.
- [ ] Kindly check on your app configuration although your app worked I encountered the error below that might cause a big problem in the future.
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Item.js (hunk 2 above)
Comment (benardop) [reply src/components/Item.js:7]: @danmainah Can you please help me with more details with regard to the first comment you gave up there with a screenshot from the terminal? How did you generate that in your local machine? Thanks

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @benardop , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️

- [ ] As the previous reviewer has earlier highlighted, kindly check on your app configuration although your app worked I encountered the error below that might cause a big problem in the future Please check out this [link](https://stackoverflow.com/questions/63912721/failed-to-load-config-react-to-extend-from) to help fix the bug 👍🏽 

image


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benardop** [comment]
@vickymarz , can you please show me the steps you followed to reproduce this error? I did request the previous review but he didn't show me the steps he followed to get this error shown above. I am very eager to know how to produce this error. Kindly assist with those steps. Thank you.

**Comment by victorokoroji** [comment]
@benardop, after starting your app on my local machine, I saw that error in my terminal. 

I attached a link for you to check out, please follow the implementations in the link

**Comment by benardop** [comment]
@vickymarz Thanks for your feedback. I'm not sure why I don't see the same error in my terminal after starting the app using the command: npm start  
Is there something I am missing?

**Review by prolajumokeoni — APPROVED** [review]
Hi @benardop, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benardop** [comment]
@prolajumokeoni Thank you for your Code Review Feedback.

**Pull request merged**